cmd/common.go

```diff
@@ -142,3 +144,12 @@ func kongVersion(config utils.KongClientConfig) (semver.Version, error) {
 	v, err := utils.CleanKongVersion(root["version"].(string))
 	return semver.ParseTolerant(v)
 }
+
+func validateNoArgs(cmd *cobra.Command, args []string) error {
+	if len(args) > 0 {
+		return errors.New("This command cannot take any positional arguments. " +
+			"Try using a flag instead.\n" +
+			"For usage information: deck <command> -h")
```

Comment:
This is very nitpicky but is important for a uniform developer experience throughout the tool.
Let's change the error to:
> positional arguments are not valid for this command, please use flags instead
Run 'deck --help' for usage.

Case-sensitivity also matters here.

Other than this, this is ready for merge.

Reply:
Will make the change and push today.

Reply:
Done